doc/manual/src/design/store/store.md

```diff
@@ -1,10 +1,21 @@
-# Nix Store
+# Store
 
-A Nix store is a collection of *store objects* referred to by *store paths*.
-Every store also has a "store directory path", which is a path prefix used for various purposes.
+A Nix store is a collection of [store objects](objects.md) and associated operations.
 
-There are many types of stores, but all of them at least respect this model.
-Some however offer additional functionality.
+These store objects can hold arbitrary data, and Nix makes no distinction if they are used as build inputs, build results, or build plans.
+
+A Nix store allows to add and retrieve store objects.
```

Comment:
Grammar nit. Perhaps
```suggestion
A Nix store allows one to store and retrieve store objects.
```
or
```suggestion
A Nix store allows storing and retrieving store objects.
```
?

Reply:
Thanks for the hint, TIL [grammatical structure of "allow"](https://english.stackexchange.com/a/60285).

I am very sensible to using the same word root too much within a given context. "Storing store objects in a store" makes my toe nails curl up. What is wrong with "add"?

```suggestion
A Nix store allows adding and retrieving store objects.
```